unabridged-works-volume-1/original/07-axe-to-the-root.adoc

```diff
@@ -1389,9 +1423,10 @@ even a foundation of eternal life, for the true seed of life;
 and are built into the living city, which is made without hands,
 and without any of the tools of man`'s wisdom.
 
+[.blurb]
 === A Brief HISTORY of the STATE of the CHURCH since the Days of the APOSTLES, with the living Seal to it; which he that hath eternal Life abiding in him can read and witness; but that Wisdom, Zeal, and Devotion, which is in the Death, cannot.
```

Comment:
Give these capitalized word the italics (with uppercase first letter) treatment as well. It might not currently render in small-caps, but I'll fix that.